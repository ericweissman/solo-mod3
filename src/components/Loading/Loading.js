import React from 'react'

const Loading = () => {
  return(
    <div className='loading'>
      <h4>Hold Your Horses!</h4>
      <p className='notes'>Whoa there, sport! Looks like the interwebs is still loading all these groovy tunes...</p>
    </div>
  )
}

export default Loading