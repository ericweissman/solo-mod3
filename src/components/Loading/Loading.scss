.loading {
  @extend .favoritesInstructions;
  .notes {
    height: 100px;
  } 
}