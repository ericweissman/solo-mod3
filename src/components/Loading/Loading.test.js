import React from 'react'
import { shallow } from 'enzyme'
import Loading from '../Loading/Loading'

describe('Loading', () => {
  let wrapper
  it('should match the correct snapshot', () => {
    wrapper = shallow(<Loading />)
    expect(wrapper).toMatchSnapshot()
  })
})