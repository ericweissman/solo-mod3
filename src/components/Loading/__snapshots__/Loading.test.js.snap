// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading should match the correct snapshot 1`] = `
<div>
  <h4>
    Hold Your Horses!
  </h4>
  <p>
    Whoa there, sport! Looks like the interwebs is still loading all these groovy tunes...
  </p>
</div>
`;
